Refresher/UI/IntegratedPatchForm.cs

```diff
@@ -113,35 +115,50 @@ protected virtual void GameChanged(object? sender, EventArgs ev)
         
         string downloadedFile = this.Accessor.DownloadFile(ebootPath);
         
+        this.LogMessage($"Downloaded EBOOT Path: {downloadedFile}");
+        if (!File.Exists(downloadedFile))
+        {
+            this.FailVerify("Could not find the EBOOT. Patching cannot continue.", clear: false);
```

Comment:
I feel like this check is useless, but whatevs.
```suggestion
            this.FailVerify("Could not find the EBOOT we downloaded. This is likely a bug. Patching cannot continue.", clear: false);
```